src/app/src/test/kotlin/org/imperial/mrc/hint/database/ProjectRepositoryTests.kt

```diff
@@ -309,8 +306,8 @@ class ProjectRepositoryTests
         assertThat(p2.sharedBy).isEqualTo(null)
         assertThat(p2.versions.count()).isEqualTo(1)
         assertThat(p2.versions[0].id).isEqualTo("v2s1")
-        assertThat(p2.versions[0].created).isEqualTo(format(ago_3h))
-        assertThat(p2.versions[0].updated).isEqualTo(format(ago_1h))
+        assertThat(toDateTime(p2.versions[0].created)).isCloseTo(ago_3h, within(1, ChronoUnit.MICROS))
```

Comment:
Another tiny DRY suggestion - maybe make a little extension function for this isCloseTo - e.g. `isSameTime`. Could wrap the cast `toDateTime` into that too.